lib/dor/services/metadata_service.rb

```diff
@@ -62,20 +40,25 @@ def label_for(identifier)
 
       def handler_for(prefix)
         handler = handlers[prefix.to_sym]
-        raise MetadataError, "Unkown metadata prefix: #{prefix}" if handler.nil?
+        raise MetadataError, "Unknown metadata prefix: #{prefix}" if handler.nil?
 
         handler
       end
 
       private
 
       def handlers
-        @handlers ||= {}
+        @handlers ||= {}.tap do |md_handlers|
```

Comment:
do we need `{}.tap ...` could this just be?
```ruby
@handlers ||= register(CatalogHandler.new, {})
```

Reply:
I guess the tap makes sense if the idea is to encourage more `register()` lines here.